src/client/common/process/environmentActivationService.ts

```diff
@@ -167,6 +167,17 @@ export class EnvironmentActivationService implements IEnvironmentActivationServi
         void envVariablesOurSelves.promise.then(() =>
             traceVerbose(`Got env vars ourselves ${getDisplayPath(interpreter?.path)} in ${stopWatch.elapsedTime}ms`)
         );
+        // If this is a conda environment and we get empty env variables from the Python extension,
+        // Then try our approach.
+        // This could happen when Python extension fails to get the activated env variables.
+        if (
```

Comment:
Came across this whilst reviewing the existing code (to see if we have telemetry for these failures)